Gopkg.lock

```diff
@@ -25,6 +25,14 @@
   revision = "792786c7400a136282c1664665ae0a8db921c6c2"
   version = "v1.0.0"
 
+[[projects]]
+  digest = "1:298c959bff24a7797f114e89aedad63d79ed0b6db6387b5a5a896fc784736c69"
+  name = "github.com/quangngotan95/go-m3u8"
```

Comment:
@lijo-jose this should not be required as we should not be dependent on this package.